activerecord/CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+*   Discard connections which may have been left in a transaction due to a thread being killed.
+
+    If a thread is killed while in a transaction, it is possible for the connection to be reused while unexpectedly left in that transaction.
+
+    Connections are now discarded in this case.
+
+    *Nick Dower*
```

Comment:
Note that there is already a changelog entry on line 216 for the previous change. I wasn't sure whether it was ok to simply update that entry. I'm also not sure this fix warrants a changelog entry.